File: Triatlon-CS/Model/Result.cs
using System;

namespace Triatlon_CSharp.Domain
{
    [Serializable]
    public class Result : Entity<int>
    {
        public Result(Participant participant, Stage stage, double score)
        {
            Participant = participant;
            Stage = stage;
            Score = score;
        }

        public Participant Participant { get; set; }

        public Stage Stage { get; set; }

        public double Score { get; set; }

        public override string ToString()
        {
            return string.Format("Id = {0} | participant = {1} | stage = {2} | score = {3}", Id, Participant.Id, Stage.Id, Score);
        }
    }
}

File: Triatlon-Java/ChatModel/src/main/java/triathlon/model/Result.java
package triathlon.model;

public class Result extends Entity<Integer>{
    private Participant participant;
    private Stage stage;
    private Double score;

    public Result(Participant participant, Stage stage, Double score) {
        this.participant = participant;
        this.stage = stage;
        this.score = score;
    }

    public Participant getParticipant() {
        return participant;
    }

    public void setParticipant(Participant participant) {
        this.participant = participant;
    }

    public Stage getStage() {
        return stage;
    }

    public void setStage(Stage stage) {
        this.stage = stage;
    }

    public Double getScore() {
        return score;
    }

    public void setScore(Double score) {
        this.score = score;
    }
}
